vendor/engines/saas/app/jobs/analytics/page_job.rb

```diff
@@ -6,6 +6,7 @@ def payload(params)
       user = params['current_user'] ? params['current_user']['id'] : "Anonymous"
       {
         user_id: user,
+        anonymous_id: params['session_id'],
```

Comment:
Should this be `anonymousId` rather than `anonymous_id`? Same question for `IdentifyUserJob`.